core-primitives/top-pool-author/src/author.rs

```diff
@@ -317,7 +317,7 @@ where
 			if let Err(e) = self.remove_top(executed_call.clone(), shard, inblock) {
 				// We don't want to return here before all calls have been iterated through,
 				// hence only throwing an error log and push to `failed_to_remove` vec.
-				error!("Error removing trusted call from top pool: {:?}", e);
+				debug!("Error removing trusted call from top pool: {:?}", e);
```

Comment:
Can you adapt the comment above accordingly?

Reply:
Yes 👍